api/naming/naming.go

```diff
@@ -9,6 +9,11 @@ import (
 	"github.com/docker/swarmkit/api"
 )
 
+const (
+	// PredefinedLabel identifies internally allocated swarm resources
+	PredefinedLabel = "com.docker.swarm.predefined"
```

Comment:
Just define the label where it is used. Don't use this package as a dumping ground. ;)

Perhaps, we can consolidate these in another PR.